event/oracle/upgrade_oracle_event.go

```diff
@@ -0,0 +1,94 @@
+package oracle
+
+import (
+	"errors"
+	oracletypes "github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/medibloc/panacea-doracle/event"
+	"github.com/medibloc/panacea-doracle/panacea"
+	log "github.com/sirupsen/logrus"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+type UpgradeOracleEvent struct {
+	service reactor
+}
+
+var _ event.Event = (*UpgradeOracleEvent)(nil)
+
+func NewUpgradeOracleEvent(s reactor) UpgradeOracleEvent {
+	return UpgradeOracleEvent{s}
+}
+
+func (e UpgradeOracleEvent) GetEventType() string {
+	return "message"
+}
+
+func (e UpgradeOracleEvent) GetEventAttributeKey() string {
+	return "action"
+}
+
+func (e UpgradeOracleEvent) GetEventAttributeValue() string {
+	return "'UpgradeOracle'"
+}
+
+func (e UpgradeOracleEvent) EventHandler(event ctypes.ResultEvent) error {
+	addressValue := event.Events[oracletypes.EventTypeUpgradeVote+"."+oracletypes.AttributeKeyOracleAddress][0]
+	service := e.service
+	queryClient := service.QueryClient()
+
+	uniqueID := service.EnclaveInfo().UniqueIDHex()
+	oracleRegistration, err := queryClient.GetOracleRegistration(addressValue, uniqueID)
```

Comment:
And If my previous comment is right, we need to query the oracle registration using the uniqueID of upgrade info, not the previous version of `doracle`.